docs/ref/settings.txt

```diff
@@ -385,6 +385,11 @@ where ``reason`` is a short message (intended for developers or logging, not for
 end users) indicating the reason the request was rejected.  See
 :doc:`/ref/csrf`.
 
+The default implementation ``django.views.csrf.csrf_failure`` takes an additional
+keyword parameter ``template_name`` that defaults to ``'csrf_fail.html'``. If a
```

Comment:
I thought about ``csrf_403.html`` (or ``403_csrf.html``?) as well and I am open for discussion here. However, I think that keeping the name the same as the setting is the clearest choice.